ui/00_message/ui/quick_party_res.xml

```diff
@@ -7,5 +7,6 @@
   <message>
     <key>quick_party_ready</key>
     <original>一定時間内に準備完了してください</original>
+    <translation>Please prepare within the period of time</translation>
```

Comment:
This probably is more like "Please complete preparations within the remaining time."